packages/cosmos_ui_components/lib/components/cosmos_wallet_edit_button.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:cosmos_ui_components/cosmos_ui_components.dart';
+import 'package:flutter/material.dart';
+
+class CosmosWalletEditButton extends StatelessWidget {
```

Comment:
this is a bit too specific component to be used in `cosmos_ui_components`. Also, this could be simply exported from figma as an asset and used inside `Image.asset` wrapped with `InkWell`, something simmilar to what `IconButton` in flutter does. We could create `CosmosImageButton` actually that allows you to pass in the path to the asset you want to use as the button

Reply:
Updated